apps/trading/client-pages/portfolio/account-history-container.tsx

```diff
@@ -95,17 +96,15 @@ const AccountHistoryManager = ({
     [pubKey]
   );
 
-  const assetsWithBalanceHistory = useAccountsWithBalanceQuery({
+  const { data: assetsWithBalanceHistory } = useDataProvider({
```

Comment:
suggestion (non blocking): This variable could use renaming now its inaccurate. `accounts` or `accountsData`